_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -1,28 +1,64 @@
 ---
-title: Changing a Deployment
+title: Updating Forseti
 order: 104
 ---
 #  {{ page.title }}
 
-This page describes how to change a Forseti Security deployment. Most changes
-to a deployment are usually to the deployment properties, such as the
-instance type or `src-path` when you want your deployment to run a certain 
-version of Forseti Security. To update your deployment, follow the process below:
+### Update local installation
+If you want to update your local version of Forseti, pull in the latest changes from
+git:
 
-  1. Edit `deploy-forseti.yaml` and update the values you want to change.
+```bash
+$ cd forseti-security
+$ git pull
+```
+
+And then run the python installation:
+
+```bash
+$ python setup.py install
+```
+
+### Change deployment properties
+If you need to change any of the properties for your Forseti deployment, such as the
+`release-version`, follow the process below:
+
+  1. If you **do not** need to update your Forseti version, continue on to step 3.
+     Sync your local git code (where you ran Deployment Manager) so you get the latest 
+     deployment templates. This is important because a newer version of Forseti might 
+     also require an updated startup script on the GCE instance.
+     
+     To get the latest version of Forseti, sync your master branch:
+     
+     ```bash
+     $ git checkout master
+     $ git pull
+     ```
+     
+  1. Check `deploy-forseti.yaml.sample` to see if there are any new properties 
```

Comment:
How do I identify if there are new properties?

Reply:
Use diff

Reply:
This would be:

```
git diff origin..HEAD~1 -- deploy-forseti.yaml.sample
```